### PR TITLE
Nanotrasen to weyland_yutani

### PR DESCRIPTION
**Pull request by cuberound**
# About the pull request

nanotrasen does not exits we have wy, leaves only one .NuclearBomb__NTIcon {
  background-image: url('../assets/bg-nanotrasen.svg');
  background-size: 70%;
  background-position: center;
  background-repeat: no-repeat;
}

# Explain why it's good for the game

anothes step away from normal ss13

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
code: changes references from nanotrasen to weyland yutani
/:cl:

### CODE REVIEW

**Comment by cuberound** [comment]
> You're messing with save files, you need to add something to update those



> You're messing with save files, you need to add something to update those

okey should be hopefuly resolved, did not handle this part of code before so double check it please

**Comment by efzapa** [comment]
dont forget to rename the centcom berets

the descs are fine, but the name is still CentCom

**Comment by cuberound** [comment]
> dont forget to rename the centcom berets
> 
> the descs are fine, but the name is still CentCom

future pr perhaps, there is a lot more centcom stuff and I do not know what to rename it to

File: code/modules/client/preferences.dm [hunk 1]
```diff
@@ -1644,7 +1644,7 @@ GLOBAL_LIST_INIT(bgstate_options, list(
 				if("nt_relation")
```
Comment (realforest2001) [inline code/modules/client/preferences.dm:1644]: This is nanotrasen too

File: maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm
Comment (Nanu308) [inline maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm:1]: map changes are just filepath changes, lgtm

**Review by Nanu308 — COMMENTED** [review]
map changes are just filepath changes, lgtm

**Review by Drulikar — APPROVED** [review]

**Pull request merged**